src/Model/BaseBlock.php

```diff
@@ -199,6 +199,26 @@ public function addChildren(BlockInterface $children): void
         $children->setParent($this);
     }
 
+    public function removeChild(BlockInterface $child): void
```

Comment:
When I made the collection inline, there was an exception asking me to add this removeChild:

```
Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException:
The add method "addChild" in class "App\Entity\SonataPageBlock" was found, but the corresponding remove method "removeChild" was not found.

  at vendor/symfony/property-access/PropertyAccessor.php:534
  at Symfony\Component\PropertyAccess\PropertyAccessor->writeProperty()
     (vendor/symfony/property-access/PropertyAccessor.php:136)
  at Symfony\Component\PropertyAccess\PropertyAccessor->setValue()
     (vendor/sonata-project/admin-bundle/src/Manipulator/ObjectManipulator.php:50)
  at Sonata\AdminBundle\Manipulator\ObjectManipulator::addInstance()
     (vendor/sonata-project/admin-bundle/src/Admin/AdminHelper.php:159)
  at Sonata\AdminBundle\Admin\AdminHelper->appendFormFieldElement()
     (vendor/sonata-project/admin-bundle/src/Action/AppendFormFieldElementAction.php:74)
  at Sonata\AdminBundle\Action\AppendFormFieldElementAction->__invoke()
     (vendor/symfony/http-kernel/HttpKernel.php:152)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:74)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php:35)
  at Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
     (vendor/autoload_runtime.php:29)
  at require_once('/usr/app/vendor/autoload_runtime.php')
     (public/index.php:7)    
```

At the following route: `https://localhost:8443/backoffice/core/append-form-field-element?_sonata_admin=sonata.page.admin.page%7Csonata.page.admin.block&_sonata_name=admin_app_sonatapagepage_sonatapageblock_edit&childId=37&elementId=s62fb36aab7a56_children&id=106&objectId=37&uniqid=s62fb36aab7a56`

I guess its caused because we added `addChild` replacing `addChildren` (before it wasn't even used that method I guess.